tests/testAutopas/tests/tuning/tuningStrategy/TuningStrategyTest.cpp

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file TuningStrategyTest.cpp
+ * @author Jan Nguyen
+ * @date 27.04.20
+ */
+
+#include "TuningStrategyTest.h"
+
+// These tests don't apply anymore since the responsibility of catching those search space errors
+// was moved to the AutoTuner.
+
+// INSTANTIATE_TEST_SUITE_P(Generated, TuningStrategyTest,
+//                         testing::ValuesIn(autopas::TuningStrategyOption::getAllOptions()),
+//                         TuningStrategyTest::PrintToStringParamName());
```

Comment:
I guess it's useful to keep in case somebody in the future thinks that we don't test `TuningStrategy`.

Reply:
What is the reason for keeping these files?

If it is as @thesamriel suggests, I'm unsure that this is a suitable reason to keep what is essentially clutter. You can always add a comment to the relevant `AutoTuner` tests, explaining that this replaces the functionality of the old `TuningStategyTest` tests.

Reply:
Initially, I kept it because I thought that there might be some common functionality to test here, but I'm not convinced that there really is any. I just deleted it: 009fe2744